README.md

```diff
@@ -86,6 +86,26 @@ static_metadata:
   * All `static_metadata` entries must have `type` specified. This specifies the Stackdriver metric type and overrides the metric type chosen by the Prometheus client.
   * If `value_type` is specified, it will override the default value type for counters and gauges. All Prometheus metrics have a default type of double.
 
+#### Dealing with recording rules
```

Comment:
Would a better title here be `#### Dealing with recorded metrics`?

Reply:
Hm.  I'm trying to keep congruent with the Prometheus terminology here; https://prometheus.io/docs/practices/rules/ and https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/ refer to `recording rules` but never to `recorded metrics`, and a google search for "prometheus recorded metric" returns those two pages first and foremost.